### ISSUE TITLE
[CI Image] Update ci_cpu to v0.80

### ISSUE THREAD

**Issue by lhutton1** [apache/tvm#9659]
- [x] S0. Reason: Update Vela to v3.2.0 (#9635)

- [x] S1. Tag of nightly build: `20211206-124834-9c4115320`. Docker hub: https://hub.docker.com/layers/tlcpackstaging/ci_cpu/20211206-124834-9c4115320/images/sha256-f8ac53e5401387d605f46bf858be41e01a3bcdda3eb4a541e4a4680d6f938fc2?context=explore

- [x] S2. The nightly is built on TVM commit: `9c4115320`. Detailed info can be found here: https://ci.tlcpack.ai/blue/organizations/jenkins/docker-images-ci%2Fdaily-docker-image-rebuild/detail/daily-docker-image-rebuild/139/pipeline/

- [x] S3. Testing the nightly image on ci-docker-staging: https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/ci-docker-staging/175/pipeline | commit : https://github.com/apache/tvm/commit/91e901becf34631bf79c6ea86306e84697c892b7

- [ ] S4. Retag `20211206-124834-9c4115320` to `v0.80`
```
docker pull tlcpackstaging/ci_cpu:20211206-124834-9c4115320
docker tag tlcpackstaging/ci_cpu:20211206-124834-9c4115320 tlcpack/ci-cpu:v0.80
docker push tlcpack/ci-cpu:v0.80
```

- [ ] S5. Check if the new tag is really there: https://hub.docker.com/u/tlcpack

- [ ] S6. Submit a PR updating the `ci-cpu` version on Jenkins

**Comment by lhutton1** [comment]
cc @leandron @areusch @manupa-arm - please let me know if there's anything I've missed :)

**Comment by manupa-arm** [comment]
I've created ci-docker-staging job : https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/ci-docker-staging/173/pipeline

**Comment by manupa-arm** [comment]
@tqchen @areusch @leandron , I personally think S4 is unnecessary here.

tlcpackstaging image tag is much more descriptive with the date and the commit hash. If we dont do S4, we might be able to just update the Jenkinsfile after ci-docker-staging job passes. WDYT ?

**Comment by leandron** [comment]
> @tqchen @areusch @leandron , I personally think S4 is unnecessary here.
> 
> tlcpackstaging image tag is much more descriptive with the date and the commit hash. If we dont do S4, we might be able to just update the Jenkinsfile after ci-docker-staging job passes. WDYT ?

I think that moving tlcpack/* docker images to use the same pattern as tlcpackstaging would be very good, because it gives an impression of how old the image is (as it is important, given we always will get the latest updates from the Ubuntu base image), as well as to which point specifically in the git repo it includes, that you can query in 1 second on your local machine.

I support this 100% - which slightly deviates from the topic of this ticket... but I think it is a great idea @manupa-arm.

**Comment by manupa-arm** [comment]
Earlier ci-docker-staging job got a timeout. Not sure why.
I've created a new one : https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/ci-docker-staging/174/pipeline

**Comment by manupa-arm** [comment]
It seems like no space in the CI node. Any ideas are welcome!

**Comment by manupa-arm** [comment]
This is blocked on https://github.com/apache/tvm/issues/9705

**Comment by leandron** [comment]
Superseded by #9762
